src/test_IndexGame.py

```diff
@@ -2,6 +2,8 @@
 import pytest
 from unittest.mock import Mock, patch
 
+from gameinit import buildingPool
```

Comment:
Hi, sorry it should be removed, it is a function initially meant to contain the building pool within the gameinit file but since we are not splitting the functions into different files, it has no use.